nailgun/entities.py

```diff
@@ -6874,12 +6874,18 @@ def __init__(self, server_config=None, **kwargs):
                 choices=('global_default_http_proxy', 'none', 'use_selected_http_proxy')
             ),
             'http_proxy_id': entity_fields.IntegerField(),
+            'deb_releases': entity_fields.StringField(),
+            'deb_errata_url': entity_fields.StringField(),
```

Comment:
@dosas Can you please reference the API doc, assuming it is publicly accessible, that references `deb_errata_url`? I could not find it at the first try.

Reply:
All I can give you right now is an open PR https://github.com/Katello/katello/pull/7961/files#diff-8a86b1f719540a398a66ebd0d9bedc010e40e5d8fb0f516464bb36e93f9adba9R26